client/layout/guided-tours/style.scss

```diff
@@ -71,7 +71,6 @@
 	animation-duration: 0.25s;
 	animation-name: guided-tours__step-slidein;
 	animation-timing-function: ease-out;
-	animation-delay: 2s;
```

Comment:
Might it be worth allowing this css rule depending on a prop, `isDelayed` or `delayAppear` for instance?
That we avoid having to add `style={ { animationDelay: '2s' } }` to each new case and could even do some logic to determine whether or not it should be delayed `isDelayed={ ! this.showImmediately }`  for example.

Reply:
@spen yeah, I explored that a bit but in the end decided against it — it would be the same as now, but with a set delay. Some tours already use a different delay, overriding the two seconds. It would make things less flexible, I think, than they need to be. And, fwiw, I just added the two seconds to existing tours so we don't change their existing behavior / as a legacy measure. Future tours might not actually need the delay or need a different one.

Reply:
Good point, Thanks for the info @lsinger :)